scss/components/_messages.scss

```diff
@@ -17,27 +17,34 @@
 
 .message {
   @include clearfix();
-  border-top: 1px solid;
+  background-size: 2rem;
```

Comment:
Properties should be ordered background-color, background-size, border-color, border-top, margin, padding